src/ontology/edits/oeo-social.omn

```diff
@@ -309,41 +310,35 @@ Class: OEO_00000158
 Class: OEO_00000213
 
     Annotations: 
-        <http://purl.obolibrary.org/obo/IAO_0000115> "Energy used for heating and cooling for buildings and industry"@en,
+        <http://purl.obolibrary.org/obo/IAO_0000115> "A heating and cooling sector is a sector that covers heating and cooling activities."@en,
+        <http://purl.obolibrary.org/obo/IAO_0000118> "H&C sector",
+        <http://purl.obolibrary.org/obo/IAO_0000233> "issue: https://github.com/OpenEnergyPlatform/ontology/issues/30
+pull request: https://github.com/OpenEnergyPlatform/ontology/pull/484",
         rdfs:label "heating and cooling sector"
```

Comment:
I think we don't need the "heating and cooling sector". I did't see this sector beeing specified seperatly anywhere so far. For my understanding its a part of the houlsehold, commercial and industry sector.

Reply:
In my interpretation the h&c sector is neither a parent nor a child class of the households, commercial and industry sectors as the h&c sector covers the heating and cooling activities in households, commercial and industry. It is orthogonal to the other sector definitions and therefore we should keep it.

Maybe this illustration helps.
![Venn_diagram_sectors](https://user-images.githubusercontent.com/36884905/87529987-0d5cd780-c690-11ea-8442-e2294db8f2a0.png)

Reply:
okay, thanks for explaining!